CodeReview/cr.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  cr.swift
+//  CodeReview
+//
+//  Created by Yeşim Daşdemir on 18.01.2022.
+//
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
+        customView.didTapButton = {
+            if let payment = self.payment {
+                self.delegate.didFinishFlow(amount: payment.amount
+                                            currency: payment.currency)
+            }
+        }
+    }
+
+    func viewDidLoad() {
```

Comment:
View lifecycle methods should be put after init methods or variables declaration.